### ISSUE TITLE
Toolbar layout at narrowing widths

### ISSUE THREAD

**Issue by cjterdi** [department-of-veterans-affairs/va.gov-cms#10516]
## Description
Review toolbar design at narrower widths per accessibility requirement edits. 

See original accessibility issue: [9920](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9920)
See original pr: [10365](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/10365)

Accessibility guidelines requires an additional title in the toolbar. The horizontal spacing to accommodate the title needs adjustment at each breakpoint to maintain legibility and accessibility.

Toolbars at different widths
![Screen Shot 2022-08-29 at 12 01 14 PM](https://user-images.githubusercontent.com/16213503/187278162-d4e34f85-f357-4a2a-a3ae-74f39b6d24b9.png)
![Screen Shot 2022-08-29 at 12 01 28 PM](https://user-images.githubusercontent.com/16213503/187278177-0817b260-a9bc-42c7-a1e2-d50ad8e52c5f.png)
![Screen Shot 2022-08-29 at 12 01 42 PM](https://user-images.githubusercontent.com/16213503/187278233-f654ce65-45f6-4337-b88b-9b56445c4aa3.png)
![Screen Shot 2022-08-29 at 12 01 54 PM](https://user-images.githubusercontent.com/16213503/187278253-ff729deb-dea5-43b7-9a6f-e2bff2ff5517.png)

## Acceptance Criteria
- [ ] Provide design alternatives for each breakpoint that also meets accessibility guidelines
- [ ] Technical feasibility review
- [ ] Identify any required change management
- [ ] Document CMS DS changes

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
@BlakeOrgan do you still see this as valid?

**Comment by BlakeOrgan** [comment]
@EWashb Potentially. There is some funkiness that happens at smaller screen sizes as pictured, but I do wonder how often our users are using the CMS at these smaller window sizes.

**Comment by EWashb** [comment]
good point @BlakeOrgan. Truly editors should only be logging in with PIV. With that in mind, we aren't catering to mobile, so what's the smallest screen size of a desktop/laptop.
